lib/mail/network/delivery_methods/stdout.rb

```diff
@@ -0,0 +1,26 @@
+require 'mail/check_delivery_params'
+require 'logger'
+
+module Mail
+  class Stdout
+    include Mail::CheckDeliveryParams
+
+    def initialize(values)
+      self.settings = default_settings.merge!(values)
+    end
+
+    attr_accessor :settings
+
+    def deliver!(mail)
+      check_delivery_params(mail)
+      settings[:logger].send(settings[:severity], mail.encoded)
+    end
+
+    private
+
+    def default_settings
+      { :logger => ::Logger.new(STDOUT), :severity => :info }
+    end
+
```

Comment:
I thought this was convention, as I had seen it in many classes (ex: [Mail::Configuration](https://github.com/mikel/mail/blob/b159e0a542962fdd5e292a48cfffa560d7cf412e/lib/mail/configuration.rb#L72)), but looking back it doesn't appear to be consistent across all. I can remove it if you would prefer.